state/azure/cosmosdb/cosmosdb_test.go

```diff
@@ -0,0 +1,78 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cosmosdb
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/dapr/components-contrib/state"
+	"github.com/stretchr/testify/assert"
+)
+
+type widget struct {
+	Color string `json:"color"`
+}
+
+func TestCreateCosmosItem(t *testing.T) {
```

Comment:
Tests basically verify we can convert `state.SetRequest` -> `CosmosItem` and that it produces the right JSON regardless of how the value was passed in.